Android.mk

```diff
@@ -208,6 +208,9 @@ endif
 ifeq ($(AUDIOPOLICY_MTK_AUDIO_ADD),1)
 LOCAL_CPPFLAGS += -DAUDIOPOLICY_MTK_AUDIO_ADD
 endif
+ifneq ($(shell cat frameworks/av/services/audiopolicy/service/AudioPolicyService.h | grep listAudioSessions),)
```

Comment:
No cat needed just call `grep -q <term> <file>.`

Reply:
How to do this? I have used same method as in USE_SERVICES_VENDOR_EXTENSION but I do not know how to test exit code in Android.mk

Reply:
I think you have to use ifeq as grep -q returns with 0 if a match was found.